CHANGELOG.md

```diff
@@ -30,8 +38,8 @@
   * Fix a bug caused by settings.json with comments.
 
 * 1.0.0  
-  * Change repository's url and issue's url.
-  * Upgrade PicGo-Core dependency: from ^1.1.5 to ^1.1.9.
+  * Change respotory's url and issus's url.
+  * Upgrade Picgo-Core dependiency: from ^1.1.5 to ^1.1.9. 
```

Comment:
It seems that you have reverted my changes, the typos come again.
Could you please undo these 'changes'?

Reply:
SURE! I have no mind to do these changes, I'll undo these.